tools/challenge-parser/parser/plugins/add-seed.test.js

```diff
@@ -72,30 +71,40 @@ describe('add-seed plugin', () => {
     expect.assertions(6);
     plugin(simpleAST, file);
     const {
-      data: { files }
+      data: { challengeFiles }
     } = file;
-    const { indexjs, indexhtml, indexcss } = files;
+    const { indexjs, indexhtml, indexcss } = challengeFiles.reduce(
+      (challengeObjs, challengeObj) => {
+        return {
+          ...challengeObjs,
+          [challengeObj.fileKey]: {
+            ...challengeObj
+          }
+        };
+      },
+      {}
+    );
```

Comment:
Yes, but this does not intimidate new-comers...

Reply:
I have to admit that I'm not the biggest fan of `reduce`.  It can be the right tool, but so often it's harder to read than the alternatives :man_shrugging: